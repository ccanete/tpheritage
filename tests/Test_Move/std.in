R test 0 0 2 2
MOVE test 5 5
EXIT