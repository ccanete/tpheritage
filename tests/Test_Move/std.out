OK
# Rectangle test of coordinates (0, 0) and (2, 2) has been created.
OK
# The object test has been moved by 5 over the x-axis and 5 over the y-axis.

# The program is being closed.